app/views/layouts/application.html.erb

```diff
@@ -15,6 +15,7 @@
         <form class="form-inline">
           <% if current_user %>
             <button class="btn btn-sm btn-outline-secondary" type="button">Logout</button>
+            <a href="/obituaries/new"><button class="btn btn-sm btn-outline-secondary" type="button">Add Obituary</button></a>
```

Comment:
Anyone know of a better way to make a button a link?

Reply:
I believe you can add the href part within the button tag, not usre if that would be any better.
`<button class="btn btn-sm btn-outline-secondary" type="button" href="/obituaries/new">Add Obituary</button>`

Reply:
I looked in the bootstrap documentation and it said the same thing Sage wrote. I am wondering if you launch `rails s`, if the button works and the problem lies in getting the test to find/click the button.

Reply:
Hmm yes, I think that was the problem. It wasn't showing up on the test. I'll try it out in `rails s` to see if that works

Reply:
Actually, It still doesn't seem to work in `rails s` when the button is `<button class="btn btn-sm btn-outline-secondary" type="button" href="/obituaries/new">Add Obituary</button>` clicking it doesn't seem to do anything...

Reply:
Thats the best methods I found, especially with get requests